### PR TITLE
feat(*): add data plane nodes page

### PR DESCRIPTION
**Pull request by Leopoldthecoder**
### Summary

Adding DP nodes page and dynamic log leveling functionality.

- [x] merge https://github.com/Kong/kong/pull/12320
- [x] use non-pr version of `@kong-ui-public/entities-data-plane-nodes`

### Issue reference

KM-114

### CODE REVIEW

File: .github/workflows/.reusable_e2e_tests_oss.yml [hunk 1]
```diff
@@ -90,6 +92,21 @@ jobs:
...
           exit $_compose_exit
 
+      - name: Start Kong hybrid
+        if: ${{ matrix.suite == 'data-plane-nodes' }}
+        timeout-minutes: 10
+        working-directory: ${{ github.workspace }}
+        env:
+          GATEWAY_IMAGE: ${{ inputs.gateway-image }}
+        run: |
+          mkdir -p hybrid
```
Comment (sumimakito) [inline .github/workflows/.reusable_e2e_tests_oss.yml:102]: Maybe we can add `chmod a+w ./hybrid` after the `mkdir`

Comment (Leopoldthecoder) [reply .github/workflows/.reusable_e2e_tests_oss.yml:102]: It worked!

File: src/pages/data-plane-nodes/List.vue [hunk 2]
```diff
@@ -0,0 +1,399 @@
...
+  return formatDate(Date.now() / 1000 + timeout, 'MMM DD, YYYY, h:mm:ss A')
+}
+
+const openLogLevelEditModal = () => {
+  modalVisible.value = true
+}
+
+const getNodeLogLevel = async (id: string) => {
+  try {
+    const response: AxiosResponse<LogLevelResponse> = await apiService.get(`clustering/data-planes/${id}/log-level`)
```
Comment (sumimakito) [inline src/pages/data-plane-nodes/List.vue:338]: very-tiny-nit: Would we use `await apiService.get<LogLevelResponse>` here?

Comment (Leopoldthecoder) [reply src/pages/data-plane-nodes/List.vue]: Updated.

**Review by sumimakito — COMMENTED** [review]
Manually tested and worked as expected. LGTM.
Let's wait until the PR package is ready.

File: src/locales/en.json [hunk 3]
```diff
@@ -205,6 +205,40 @@
...
+        "last.seen": "Last Seen",
+        "sync.status": "Sync Status",
+        "cert.expires.at": "Cert Expires At",
+        "log.level": "Log Level",
+        "labels": "Labels"
+      },
+      "empty": {
+        "title": "No Data Plane Nodes",
+        "message": "There is no Data Plane Node connected to this Control Plane",
+        "button": "Install Plugin"
```
Comment (raoxiaoyan) [inline src/locales/en.json:226]: What does this mean?

**Review by raoxiaoyan — COMMENTED** [review]
Here, it would be better to show the `hostname` of the DP instead of `ID`
image

File: src/locales/en.json (hunk 3 above)
Comment (Leopoldthecoder) [reply src/locales/en.json]: Deleted.

**Review by raoxiaoyan — APPROVED** [review]
LGTM and local manual test.

**Pull request merged**